src/Styleguide/Components/AuctionResultItem.tsx

```diff
@@ -0,0 +1,182 @@
+import React from "react"
+import { Responsive } from "../Utils/Responsive"
+import { Flex } from "../Elements/Flex"
+import { Image } from "../Elements/Image"
+import { Serif } from "@artsy/palette"
+import styled from "styled-components"
+import { themeGet } from "styled-system"
+import TextLink from "../../Components/TextLink"
+
+const ScaledArtworkImage = props => (
+  <Flex height={props.height} justifyContent="center">
+    <Image {...props} />
+  </Flex>
+)
+
+export interface Dimensions {
+  in: string
+  cm: string
+}
+
+export interface AuctionResultItemProps {
+  imageUrl: string
+  title: string
+  date: string
+  dimensions: Dimensions
+  organization: string
+  auctionDate: string
+  salePrice: string
+  estimate: string
+  description: string
+}
+
+export class AuctionResultItem extends React.Component<AuctionResultItemProps> {
+  render() {
+    return (
+      <Responsive>
+        {({ xs, sm }) => {
+          if (xs) return <ExtraSmallAuctionResultItem {...this.props} />
+          else if (sm) return <SmallAuctionResultItem {...this.props} />
+          return <LargeAuctionResultItem {...this.props} />
```

Comment:
@mzikherman - It looks like there's `@l`, `@m` and `@s`[for this page](zpl://project?pid=5b113f06463238c438431ea3).

Regarding your question in the description, i think at the small breakpoint the left column stops growing and is fixed at a fixed width (slightly larger than the small image), and the middle / right columns stretch.